package.json

```diff
@@ -68,14 +68,14 @@
     "clean": "rm -rf dist && shx mkdir -p dist/popup",
     "watch": "npm-run-all --parallel watch:*",
     "copy:locales": "node pontoon-to-webext",
-    "copy:assets": "shx cp -r src/manifest.json src/icons dist/ && svgo src/icons -o dist/icons && shx cp -r src/popup/*.html src/popup/FiraSans-Regular.* dist/popup/",
```

Comment:
OK, tweaked this slightly to `cp -r src/manifest.json src/icons dist/ && svgo dist/icons --quiet`.
That should copy all the "src/icons/\*" files (PNG and SVG) into "dist/icons", and then just run `svgo` optimizer on all the dist/icons/\*.svg files and overwrite them in place.